app/ui/docker/Dockerfile

```diff
@@ -1,60 +1,27 @@
-FROM docker.io/centos:7
-
-ENV NGINX_VERSION 1.13.4-1.el7
+FROM scratch
```

Comment:
really ? I always find it super hard to debug, but you might have other experiences here.

Reply:
I find debugging go binaries hard in a container even with a centos base layer. By avoiding the base layer you avoid having to rebuild due to base layer CVEs.

Reply:
I'm fine with this but we should double check with Darren that scratch images are explicitly allowed as I remember darkly that there were some restrictions (but can be wrong here). What was Darrens GitHub handle again ?